src/app/offers/offer-details/offer-details.component.html

```diff
@@ -30,183 +30,190 @@
             </div>
 
             <div class="offer-right first"
-             [class.add-padding]="states.project.documents.length === 0 &&
+                 [class.add-padding]="states.project.documents.length === 0 &&
              states.project.news.length === 0">
```

Comment:
Should this 2 lines be aligned?

Reply:
You are right. Intellij is sometimes stupid about auto-formatting.